docs/folder-structure.md

```diff
@@ -138,6 +138,7 @@ Additional values passed:
 * `flake` maps to `inputs.self`.
 * `perSystem`: contains the packages of all the inputs, filtered per system.
     Eg: `perSystem.nixos-anywhere.default` is a shorthand for `inputs.nixos-anywhere.packages.<system>.default`.
+* `hostname` as per the host directory name.
```

Comment:
In the packages we call this the `pname`. If we could find a common nomenclature then it would make it transferrable knowledge. Maybe `folderName` or `entryName`?

Reply:
Interesting feedback, I had thought about naming it `hostName` because that's the name in `networking.hostName` and would make it slightly less annoying to `inherit`.
But didn't even think about this possibly being transferable knowledge. I think specialArgs are random enough in practice that I have to look them up or just try to use them anyway.

IMO `folderName` isn't ideal, because for packages it's typically a file (deliberately not getting into the `folder` vs `directory` terminology debate here ;))

`entryName` would work, but if we are that generic: Why not just `name`? Would that work for you?

Reply:
I vote for hostName fwiw

Reply:
Ah, just noticed that i forgot to change pname -> name as well. I'll happily do so once @zimbatm confirms that name is fine with him :)

Reply:
Ok let's do `hostName`. `name` is too generic.

Reply:
Done